src/ProtocolHandler/HTTP/Requests/IRequest.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Chorizo.ProtocolHandler.HTTP.Requests
+{
+    public interface IRequest
```

Comment:
I don't think this has any behavior, so it can just be a struct-ish kind of thing.